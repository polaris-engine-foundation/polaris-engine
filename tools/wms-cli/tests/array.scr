func main() {
    a[0] = 1;
    a[1] = -2;
    print(a[0] + a[1]);

    b = 1;
    b["key1"] = 2;
    b["key2"] = "value2";
    b[100] = -123;
    print(b);
    print("size=" + size(b));

    c = b;

    remove(b, "key1");
    print("b=" + size(b));
    print(b);

    remove(b, "key2");
    print("b=" + size(b));
    print(b);

    remove(b, 100);
    print("b=" + size(b));
    print(b);

    print("c=" + size(c));
    print(c);
}
