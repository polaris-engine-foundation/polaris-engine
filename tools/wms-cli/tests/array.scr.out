-1
[100:-123, "key2":"value2", "key1":2]
size=3
b=2
[100:-123, "key2":"value2"]
b=1
[100:-123]
b=0
[]
c=0
[]
